samples/system-test/requesterPays.test.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const fs = require(`fs`);
```

Comment:
Just curious-- is this the new thing to do, using the ticks in place of all string quote marks?

Reply:
In our tests, we found ourselves doing a lot of string interpolation, as well as putting both single and double quotes in strings, so we found it easier to simply use the ES2015 string templates throughout the tests.

I've seen JS devs starting to use the string templates all over, calling them the new, better JavaScript strings.

Reply:
Cool, thanks!